pkgs/development/libraries/libisds/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, expat, gpgme, libgcrypt, libxml2 }:
+
+stdenv.mkDerivation rec {
+  name = "libisds-${version}";
+  version = "0.10.8";
+
+
+
+  src = fetchurl {
+    url = "http://xpisar.wz.cz/libisds/dist/${name}.tar.xz";
+    sha256 = "1av6i7acqrsl054mm6hbz20qcnbmxmzbax942c31576f00vwcqbq";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = [ expat gpgme libgcrypt ];
+  propagatedBuildInputs = [ libxml2 ];
```

Comment:
Have you tried just having this be a build input?

Reply:
If that doesn't work, I expect the proper thing to do is use makeWrapper to make the library available where needed.

Reply:
It should be fine. It is in the `Requires` section of pc file.